src/main/java/tasks/Deadline.java

```diff
@@ -0,0 +1,46 @@
+package tasks;
+
+import java.time.LocalDateTime;
+import parser.DateTimeHandler;
+
+/**
+ * The {@code Deadline} class represents a task with a specific deadline. It extends the
+ * {@code Task} class and includes additional information about the due date and time for
+ * the task.
+ */
+public class Deadline extends Task {
+
+    private final LocalDateTime time;
```

Comment:
Perhaps a more specialised name for the time variable could be more appropriate?